bigbluebutton-html5/imports/ui/components/timer/panel/component.tsx

```diff
@@ -404,65 +490,162 @@ const TimerPanel: React.FC<TimerPanelProps> = ({
+                      {intl.formatMessage(intlMessages.minutes)}
+                    </Styled.TimeUnitLabel>
+                  </Styled.TimeUnitContainer>
+                  <Styled.TimeUnitContainer>
+                    <Styled.TimerInput
+                      type="number"
+                      readOnly={running}
+                      disabled={running}
+                      value={String(seconds).padStart(2, '0')}
+                      max="59"
+                      min="0"
+                      step="1"
+                      onChange={handleSecondsChange}
+                      onFocus={() => setFocusedUnit('seconds')}
+                      ref={secondsInputRef}
+                      data-test="secondsInput"
+                      isSelected={!running && focusedUnit === 'seconds'}
+                    />
+                    <Styled.InputArrows disabled={running} aria-hidden={running}>
+                      <Styled.InputArrowButton
+                        type="button"
+                        aria-label={`${intl.formatMessage(intlMessages.secondsAbbr)} +1`}
+                        disabled={running}
+                        onClick={() => incUnit('seconds')}
+                      >
+                        ▲
+                      </Styled.InputArrowButton>
+                      <Styled.InputArrowButton
+                        type="button"
+                        aria-label={`${intl.formatMessage(intlMessages.secondsAbbr)} -1`}
+                        disabled={running}
+                        onClick={() => decUnit('seconds')}
+                      >
+                        ▼
+                      </Styled.InputArrowButton>
+                    </Styled.InputArrows>
+                    <Styled.TimeUnitLabel>
+                      {intl.formatMessage(intlMessages.seconds)}
+                    </Styled.TimeUnitLabel>
+                  </Styled.TimeUnitContainer>
+                </Styled.TimeInputGroup>
+                {/* External +/- buttons removed; using inline arrows inside each input */}
+                <Styled.TimerAddsRow>
+                  <Styled.TimerAddButton
+                    onClick={() => changeTime(30)}
                     disabled={running}
-                    value={String(hours).padStart(2, '0')}
-                    maxLength={2}
-                    max={MAX_HOURS}
-                    min="0"
-                    onChange={handleHoursChange}
-                    onFocus={() => setFocusedUnit('hours')}
-                    data-test="hoursInput"
-                    isSelected={!running && focusedUnit === 'hours'}
-                  />
-                  <Styled.TimeInputColon>:</Styled.TimeInputColon>
-                  <Styled.TimerInput
-                    type="number"
-                    readOnly={running}
+                    aria-label="+ 30 seconds"
+                    data-test="add30s"
+                  >
+                    +0:30
```

Comment:
Could we make this section dynamic instead of hardcoding each button?
For example, define an array of time increments and map over it to generate the buttons. That would make it easier to maintain or extend in the future.